media-push/develop/integration-best-practices.mdx

```diff
@@ -73,105 +71,42 @@ the following two ways to implement it:
-|[`api-cn-east-1.agora.io`](`http://api-cn-east-1.agora.io`)               |East China            |
-|[`api-cn-north-1.agora.io`](`http://api-cn-north-1.agora.io`)             |North China           |
+* If resources are insufficient, then tasks started later will fail.
+* Media push supports up to three-way isolation, That is, `my-taskagora-isolate1`,`my-task-agoraisolate-2`, and `my-taskagora-isolate-3` can be separated.
 
+<SwitchDomainName/>
 
 ## Create RTMP Converter
 
 When creating a converter with a `create` call, pay attention to the following:
 
 - Create the converter by region parameter:
-    - The region you set must be in the same area as your CDN origin station. For example, if your CDN origin is located in US,
-    set region to `na`.
+    - The region you set must be in the same area as your CDN origin station. For example, if your CDN origin is located in US, set region to `na`.
     - Pass the region value in lowercase.
-- Agora recommends that you assign a value to the `X-Request-ID` field in the request header.
-The Agora server returns an `X-Custom-request-ID` field in the response header for troubleshooting purposes.
-- Set UID or account as the user name for the converter. Do not set these two fields at the same time. Ensure that each
-converter has a unique user name within the channel.
-- To avoid repeated media streaming due to repeated creation of multiple converters, it is recommended to use the name field to
-manage converters under a specific project:
-    - At the same time, converters with the same names cannot exist in a
-    project. If you attempt to create a converter with the same name, you will receive the `409` (Conflict) status code.
--   Agora recommends that you assign the name using a combination of the channel name and a converter property. For example,
-`show68_horizontal` and `show68_vertical` would represent a horizontal and a vertical layout respectively.
+- Agora recommends that you assign a string value to the `X-Request-ID` field in the request header. The Agora server returns an `X-Custom-request-ID` field in the response header for troubleshooting purposes.
+- Set UID or account as the user name for the converter. Do not set these two fields at the same time. Ensure that each converter has a unique user name within the channel.
+- To avoid repeated media streaming due to repeated creation of multiple converters, it is recommended to use the name field to manage converters under a specific project.    At the same time, converters with the same names cannot exist in a project. If you attempt to create a converter with the same name, you will receive the `409` (Conflict) status code.
+-   Agora recommends that you assign the name using a combination of the channel name and a converter property. For example, `show68_horizontal` and `show68_vertical` would represent a horizontal and a vertical layout respectively.
```

Comment:
These documents use "will" multiple times. However, future tense in general should be avoided: https://developers.google.com/style/tense

Reply:
Removed all occurrences of "will"